### PR TITLE
Swift Package Manager

### PR DESCRIPTION
**Pull request by gjurzak**
Added SPM manifest file.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "RxMKMapView",
+    platforms: [.iOS(.v10)],
```
Comment (sjrmanning) [inline Package.swift]: Is there any reason this is `v10` rather than `v8` as specified in the Podfile?

**Review by sjrmanning — CHANGES_REQUESTED** [review]
Hey, thanks for this! Sorry it's taken a while to notice. Just one minor comment about iOS platform version.

File: Package.swift (hunk 1 above)
Comment (gjurzak) [reply Package.swift]: hmm i think any reason. will set v8 + bump rx swift

**Comment by gjurzak** [comment]
@sjrmanning Fixed :)

**Comment by sjrmanning** [comment]
Thanks!

**Pull request merged**